internal/pkg/instrumentation/bpf/google.golang.org/grpc/client/probe.go

```diff
@@ -138,6 +154,7 @@ func convertEvent(e *event) []*probe.SpanEvent {
 			SpanContext:       &sc,
 			ParentSpanContext: pscPtr,
 			TracerSchema:      semconv.SchemaURL,
+			Status:            probe.Status{Code: codes.Code(int(e.StatusCode))},
```

Comment:
This should only set the status for certain error status codes: https://github.com/open-telemetry/semantic-conventions/blob/5eb37f706894718493af7a668971c651acb80812/docs/rpc/grpc.md#grpc-status

And we will need to set it to `codes.Error`.

Reply:
Thanks, updated this. Looks like client spans are `codes.Error` for anything besides success which is easy here at least hah